source/image-handler/index.js

```diff
@@ -20,20 +20,19 @@ exports.handler = async (event) => {
     const imageHandler = new ImageHandler();
     try {
         const request = await imageRequest.setup(event);
-        console.log(request);
-        const processedRequest = await imageHandler.process(request);
+        const { contentType, body } = await imageHandler.process(request);
         const response = {
             "statusCode": 200,
-            "headers" : getResponseHeaders(),
-            "body": processedRequest,
+            "headers": getResponseHeaders(false, contentType),
```

Comment:
We now use false to indicate there is no error, we use the contentType we got from the imageHandler